slycot/math.py

```diff
@@ -70,6 +70,361 @@ def mc01td(dico,dp,p):
     return out[:-2]
 
 
+def mb03vd(n, ilo, ihi, A):
```

Comment:
The idea is to leave it open and let the Fortran routine populate the `[:n,:n]` part just as it is in the docstring. The routines in analysis.py do it the same way. Infering from the size of input `A` is also possible but then restrains n>=1 instead of n>=0 like in the Fortran routine. I don't think that that would change anything substantial.

Reply:
HQ and Tau are probably fine, and the caller is anyway not obliged to use the same names as in the docstrings.